Adds procedure to create private cluster with private image registry (#69087)
openshift/openshift-docs: installing/installing_azure/installing-azure-private.adoc

// Additional steps for the Cloud Credential Operator utility (`ccoctl`)
include::modules/cco-ccoctl-install-creating-manifests.adoc[leveloffset=+3]

include::modules/installing-private-image-registry-private-azure.adoc[leveloffset=+1]

[role="_additional-resources"]
.Additional resources

> * See xref:../../installing/installing_azure/installing-azure-account.adoc#minimum-required-permissions-ipi-azure_installing-azure-account[Required Azure permissions for installer-provisioned infrastructure] for the list of permissions needed to create a private storage endpoint.

Reviewer: Rewrite as "For the list of permissions needed to create a private storage endpoint, see ...".

> * See xref:../../installing/installing_azure/installing-azure-account.adoc#minimum-required-permissions-ipi-azure_installing-azure-account[Required Azure permissions for installer-provisioned infrastructure] for the list of permissions needed to create an image registry storage account private endpoint.

Reviewer 2: ```suggestion
* See xref:../../installing/installing_azure/installing-azure-account.adoc#minimum-required-permissions-ipi-azure_installing-azure-account[Required Azure permissions for installer-provisioned infrastructure] for the list of permissions needed to create a private storage endpoint.
```

Author: Fixed!
